vocab.py

```diff
@@ -22,7 +18,7 @@ class ValidJobName(Enum):
 
     COURSE_INVENTORY = 'course_inventory.inventory.run_course_inventory'
     MIVIDEO = 'mivideo.mivideo_extract.main'
-    CANVAS_ZOOM_MEETINGS = 'online_meetings.canvas_zoom_meetings.main'
+    CANVAS_ZOOM_MEETINGS = 'lti_placements.canvas_placements.main'
```

Comment:
Okay, I'm giving this a test run now. Do you think we should change the name of the job? CANVAS_LTI maybe? The current one doesn't cover everything you're doing now.

Reply:
If we do, I think we need to update the controlled vocabulary in `env_schema.json` for `JOB_NAMES`.

Reply:
Yeah I can update this.